CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.61.0
+
+- Packit can now correctly authenticate with Bodhi 6 and therefore create Bodhi updates. 🚀 (#1746)
+- Packit now requires Python 3.9 or later. (#1745)
```

Comment:
Hm, good question :D Probably not.... How should we handle this? :thinking: Anyway, I am still waiting for a fix in specfile (the failing epel9 test)

Reply:
Or maybe we can just release normally, since all current stable releases already have 3.10. WDYT?

Reply:
I don't see any reason why we couldn't push this to stable Fedoras, that was the plan since the start. There will be no change for users, they've been using packit built for Python 3.10 and they will continue to do so.

Reply:
The only distro we can't push this into is EPEL 8, but EPEL 8 is already excluded from downstream automation, it won't be getting new packit versions anymore.